server/bin/test-bin/test_logger_type.py

```diff
@@ -10,6 +10,7 @@
 _NAME_ = "pbench-logger-test"
 cfg_name = os.environ["_PBENCH_SERVER_CONFIG"]
 logdir = os.environ["LOGSDIR"]
+assert os.path.isdir(logdir), f"LOGSDIR={logdir}"
```

Comment:
That message seems a little cryptic -- it doesn't mention that the indicated directory doesn't actually exist....

Reply:
Fixed.